File: AteliwareGitHub.Server/EFModels/GitRepos.cs
using System;
namespace AteliwareGitHub.Server
{
    public class GitRepos
    {
        public int id { get; set; }
        public int gitID { get; set; }
        public string name { get; set; }
        public int stars { get; set; }
        public string description { get; set; }
        public string url { get; set; }
        public DateTime SearchDate { get; set; }
        public string language { get; set; }
    }
}

File: backend/models/gitrepos.js
module.exports = (sequelize, DataTypes) => {
  const GitRepos = sequelize.define('GitRepos', {
    name: DataTypes.STRING,
    language: DataTypes.STRING,
    description: DataTypes.STRING,
    user: DataTypes.STRING,
    userImage: DataTypes.STRING,
    created: DataTypes.DATE,
    updated: DataTypes.DATE,
    stars: DataTypes.INTEGER,
    watchs: DataTypes.INTEGER,
    size: DataTypes.INTEGER,
    issues: DataTypes.INTEGER,
    url: DataTypes.STRING,
  });
  return GitRepos;
};
